tests/test_army_pay_decree_1503.py

```
@@ -0,0 +1,457 @@
+    assert payload["purpose"] == "补饷"
+    assert payload["target_kind"] == "army"
+    assert payload["target_id"] == "guanning"
+    assert int(payload["amount"]) == 15
+    assert payload["account"] == "国库"
+
+
+def test_army_pay_missing_fields_fail_loud_at_admission(game):
+    """字段缺失 fail-loud：不猜散文、不成案、不落账。"""
+    db, state, content = game
+    treasury_before = int(state.metrics["国库"])
+    with pytest.raises(ValueError, match="拨饷|amount|account|target"):
```

Ruff (0.16.1)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)